lib/jazzy/sourcekitten.rb

```diff
@@ -45,7 +46,10 @@ def self.make_doc_urls(docs, parents)
 
     # Run sourcekitten with given arguments and return STDOUT
     def self.run_sourcekitten(arguments)
-      bin_path = Pathname(__FILE__).parent + '../../bin'
+      bin_path = Shellwords.shellescape Pathname(__FILE__).parent + '../../bin'
```

Comment:
Really this just shows that we should be launching processes in a more proper way (I.e. popen3)

-Samuel E. Giddins

On Jan 15, 2015, at 1:54 AM, James Barrow <notifications@github.com> wrote:

In lib/jazzy/sourcekitten.rb:

> @@ -45,7 +46,10 @@ def self.make_doc_urls(docs, parents)
>  
>      # Run sourcekitten with given arguments and return STDOUT
>      def self.run_sourcekitten(arguments)
> -      bin_path = Pathname(__FILE__).parent + '../../bin'
> +      bin_path = Shellwords.shellescape Pathname(__FILE__).parent + '../../bin'
Without it Jazzy always fails for me with No such file or directory error showing the bin_path only up to the first space.

—
Reply to this email directly or view it on GitHub.